### PR TITLE
feat(err): Store symbol sets

### PR DESCRIPTION
**Pull request by oliverb123**
After we fetch a symbol set, we should stick it in s3, so we're not hitting customer infra, and so we can do things like invalidate saved frame resolution results if a customer gives us a new symbol set manually.

### CODE REVIEW

**Review by daibhin — COMMENTED** [review]
Maybe a few tests to verify the flow?

File: rust/cymbal/src/config.rs [hunk 1]
```diff
@@ -38,6 +38,12 @@ pub struct Config {
 
     #[envconfig(default = "100000000")] // 100MB - in prod, we should use closer to 1-10GB
     pub symbol_store_cache_max_bytes: usize,
+
+    #[envconfig(default = "symbol_sets")]
+    pub ss_bucket: String,
+
+    #[envconfig(default = "sets")]
+    pub ss_prefix: String,
```
[the remark this answers is not in the document]
Comment (oliverb123) [reply rust/cymbal/src/config.rs:46]: I think that's fine - these are just the config defaults, we can change them to align or we can just make sure to set it as it should be in production

File: rust/cymbal/src/symbol_store/saving.rs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    pool: PgPool,
+    bucket: String,
+    prefix: String,
+}
+
+// A record of an attempt to fetch a symbol set. If it succeeded, it will have a storage pointer
+pub struct SymbolSetRecord {
+    id: Uuid,
+    team_id: i32,
+    // "ref" is a reserved keyword in Rust, whoops
```
[the remark this answers is not in the document]
Comment (oliverb123) [reply rust/cymbal/src/symbol_store/saving.rs:25]: Indeed - nbd though, thank god for `as`

**Review by daibhin — APPROVED** [review]

File: rust/.cargo/config.toml [hunk 3]
```diff
@@ -1,4 +1,4 @@
 [env]
 # Force SQLX to run in offline mode for CI. Devs can change this if they want, to live code against the DB,
 # but we use it at the workspace level here to allow use of sqlx macros across all crates
-SQLX_OFFLINE = "true"
+SQLX_OFFLINE = "false"
```
[the remark this answers is not in the document]
Comment (oliverb123) [reply rust/.cargo/config.toml]: Yes, done

**Review by daibhin — APPROVED** [review]

**Pull request merged**